### DISCUSSION TITLE
Repository coordinates for the JAR?

### DISCUSSION THREAD

**Discussion by crimeminister** [kuzudb/kuzu#4127] [Q&A] [answered]
I'd like to be able to pull the Kùzu JAR file from a repository. Is it (or will it be) published somewhere like Maven Central, and if so what are the coordinates?

**Comment by prrao87** [comment]
Hi @crimeminister, @mewim has been thinking about doing this for a while now, we just needed some traction from the community. I think it's worth keeping this open till we can have this prioritized on our end. Thanks!

**Comment by mewim** [comment, marked as answer]
It is now available at https://central.sonatype.com/artifact/com.kuzudb/kuzu

**Reply by crimeminister** [reply]
Thanks!
